### PR TITLE
Minor improvements to shared React components

### PR DESCRIPTION
**Pull request by shivam-kapila**
- [x] Improve card shadows
- [x] Improve card radius
- [x] Add spinner icon
- [x] Add spinner text
- [x] Update pills
- [x] Make navbar sticky
- [x] Fix charts height
- [x] Fix heatmap height

### CODE REVIEW

**Comment by MonkeyDo** [comment]
A chat we had about adding text in the loader:
https://chatlogs.metabrainz.org/brainzbot/metabrainz/2020-08-06/?msg=4639746&page=2

File: listenbrainz/webserver/static/js/src/components/Loader.tsx [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
-      <FontAwesomeIcon icon={faSpinner as IconProp} size="4x" spin />
+    <div className={`text-center ${className || ""}`} style={style} {...rest}>
+      <Spinner type="Oval" color="#cccccc" height={30} width={30} />
+      {!loaderText ? (
+        <small>
+          <p className="text-muted mt-5">{loaderText}</p>
+        </small>
+      ) : (
+        <></>
+      )}
```
Comment (ishaanshah) [inline listenbrainz/webserver/static/js/src/components/Loader.tsx:40]:
```suggestion
      {loaderText && (
        <small>
          <p className="text-muted mt-5">{loaderText}</p>
        </small>
      )}
```

File: listenbrainz/webserver/static/js/src/stats/HeatMap.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default function Heatmap(props: HeatMapProps) {
       width={width}
       height={height}
       enableLabels={false}
-      padding={1}
+      padding={4}
```
Comment (ishaanshah) [inline listenbrainz/webserver/static/js/src/stats/HeatMap.tsx:39]: IMO the graph looks better with padding set to 1. It doesn't look good on mobile view and the ticks look misaligned.

File: listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx [hunk 3]
```diff
@@ -104,19 +104,12 @@ export default class UserTopEntity extends React.Component<
...
-          height: 550,
+          minHeight: 550,
           marginTop: 20,
-          display: "flex",
-          flexDirection: "column",
-          alignItems: "center",
         }}
       >
-        <h3 className="capitalize-bold">Top {entityTextOnCard}</h3>
-        <Loader
```
Comment (ishaanshah) [inline listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx:115]: I am not sure what exactly is the cause of the issue but the alignment is incorrect for the entity list.
PR -
2020-08-14-171515_381x567_scrot
Production -
2020-08-14-171505_389x567_scrot

File: listenbrainz/webserver/static/js/src/stats/HeatMap.tsx (hunk 2 above)
Comment (shivam-kapila) [reply listenbrainz/webserver/static/js/src/stats/HeatMap.tsx:39]: okk

File: listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx (hunk 3 above)
Comment (shivam-kapila) [reply listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx:115]: Caught it. Flexbox helping us

**Pull request merged**